bigquery_etl/cli/glean_usage.py

```diff
@@ -109,3 +110,14 @@ def generate(project_id, output_dir, parallelism, exclude, only, app_name):
                 partial(table.generate_per_app, project_id, output_dir=output_dir),
                 app_info,
             )
+
+    # generate per-app events_unnested views
+    with ThreadPool(parallelism) as pool:
+        pool.map(
+            partial(
+                events_unnested.EventsUnnestedTable().generate_per_app,
+                project_id,
+                output_dir=output_dir,
+            ),
+            app_info,
+        )
```

Comment:
Can we include `EventsUnnestedTable` in `GLEAN_TABLES` and avoid this special case? It seems fine to have the `generate_per_app_id` be a no-op and let it run. Are there other reasons to have this be a special case?